### DISCUSSION TITLE
Link to internal category or article

### DISCUSSION THREAD

**Discussion by amuijs** [templaza/astroid-framework#693] [General]
Is it possible to link to an article or category?

**Comment by Chacapamac** [comment]
To find the URL link of an article or a category in Joomla, you can follow these steps:

    For an Article:
        Navigate to Content > Articles in your Joomla administration panel.
        Locate and open the article for which you want the URL.
        If you have already created a menu item for this article, the URL is typically formed by the base URL of your site followed by the alias of the menu item. For example: http://www.yoursite.com/menu-item-alias.
        If the article is featured or belongs to a category that is displayed on a page, the URL will be the one of that page.

    For a Category:
        Go to Menus > [Your Menu] where the category is listed.
        Find the menu item that links to the category view and check its alias.
        The URL will be formed by the base URL of your site followed by the alias of the menu item. For example: http://www.yoursite.com/category-menu-item-alias.

Remember, if the article or category is not linked to a menu item, it may not be directly accessible via a URL. [In such cases, you might need to create a new menu item that links to the article or category to make it accessible](https://forum.joomla.org/viewtopic.php?t=1007259)[1](https://forum.joomla.org/viewtopic.php?t=1007259)[2](https://docs.joomla.org/J4.x:Article_Links/en)[3](https://forum.joomla.org/viewtopic.php?t=258720).
